core/trino-parser/src/main/java/io/trino/sql/tree/DefaultTraversalVisitor.java

```diff
@@ -683,6 +683,13 @@ protected Void visitExplain(Explain node, C context)
         return null;
     }
 
+    @Override
+    protected Void visitShowStats(ShowStats node, C context)
```

Comment:
Please add unit tests.

Reply:
eg in `TestParameterExtractor` but we also need an integration tests, perhaps in  `TestShowStats`, although I am not sure the QueryRunner supports PREPARE + EXECUTE, does it?

Reply:
I can write both unit and integration tests.